README.md

```diff
@@ -33,27 +33,30 @@ section.
 ```js
 const Koa = require('koa')
 const app = new Koa()
-
+const compose = require('koa-compose')
 const router = require('koa-router')()
-const WebMonetization = require('koa-web-monetization')
-const monetization = new WebMonetization()
-
-// This is the SPSP endpoint that lets you receive ILP payments.  Money that
-// comes in is associated with the :id
-router.get('/pay/:id', monetization.receiver())
+const KoaWebMonetization = require('koa-web-monetization')
+const monetizer = new KoaWebMonetization()
 
 // This endpoint charges 100 units to the user with :id
 // If awaitBalance is set to true, the call will stay open until the balance is sufficient. This is convenient
 // for making sure that the call doesn't immediately fail when called on startup.
-router.get('/content/:id/:content_id/', monetization.paid({ price: 100, awaitBalance: true }), async ctx => {
-  // load content by :content_id
+router.get('/content/', async ctx => {
+
+  await ctx.awaitBalance(100)
+  ctx.spend(100)
+  // load content
 })
 
 router.get('/', async ctx => {
   // load index page
 })
 
+router.get('/client.js', async ctx => {
+  ctx.body = monetizer.serveClient()
+}
 app
+  .use(compose([monetizer.mount(), router.middleware()]))
```

Comment:
Still not so sure we should be using compose; can't we fix that issue with ctx.state?